tests/main.test.ts

```diff
@@ -270,6 +270,11 @@ it(`should refuse to run a different package manager within a configured project
     process.env.COREPACK_ENABLE_STRICT = `0`;
 
     try {
+      await expect(runCli(cwd, [`yarn`, `--version`])).resolves.toMatchObject({
+        stdout: `1.0.0\n`,
+        stderr: ``,
+        exitCode: 0,
+      });
       await expect(runCli(cwd, [`pnpm`, `--version`])).resolves.toMatchObject({
```

Comment:
I'm quite lost why tests failed in CI. I'm able to successfully run the tests in my local machine (and it all passes) 😅

Reply:
Yeah I had to fight with the CI a bit, it's finally green now 😅